packages/admin-ui/ui/public/locales/th/translation.json

```diff
@@ -0,0 +1,2011 @@
+  "price-overrides-apply-overrides-on-selected-variants": "ใช้การแทนที่กับตัวแปรที่เลือก",
+  "price-overrides-apply-on-all-variants": "ใช้ได้กับทุกรุ่น",
+  "price-overrides-prices": "ราคา",
+  "price-overrides-cancel": "ยกเลิก",
+  "price-overrides-save-and-close": "บันทึกและปิด",
+  "price-overrides-show-regions": "แสดงภูมิภาค",
+  "product-table-products": "สินค้า",
+  "product-table-copy-success": "สำเร็จ",
+  "product-table-copy-created-a-new-product": "สร้างผลิตภัณฑ์ใหม่",
+  "product-table-copy-error": "ข้อผิดพลาด",
+  "product-table-delete-product": "ลบสินค้า",
+  "product-table-confirm-delete": "คุณแน่ใจหรือไม่ว่าต้องการลบผลิตภัณฑ์นี้",
+  "product-table-edit": "แก้ไข",
+  "product-table-unpublish": "ยกเลิกการเผยแพร่",
+  "product-table-publish": "เผยแพร่",
+  "product-table-draft": "ร่าง",
+  "product-table-published": "เผยแพร่",
+  "product-table-success": "สำเร็จ",
+  "product-table-successfully-unpublished-product": "ยกเลิกการเผยแพร่ผลิตภัณฑ์เรียบร้อยแล้ว",
+  "product-table-successfully-published-product": "เผยแพร่ผลิตภัณฑ์เรียบร้อยแล้ว",
+  "product-table-error": "ข้อผิดพลาด",
+  "product-table-duplicate": "ทำซ้ำ",
+  "product-table-delete": "ลบ",
+  "product-table-proposed": "เสนอ",
+  "product-table-published-title": "เผยแพร่",
+  "product-table-rejected": "ถูกปฏิเสธ",
+  "product-table-draft-title": "ร่าง",
+  "product-table-name": "ชื่อ",
+  "product-table-collection": "ของสะสม",
+  "product-table-status": "สถานะ",
+  "product-table-availability": "ความพร้อม",
+  "product-table-inventory": "รายการสิ่งของ",
+  "product-table-inventory-in-stock-count_one": " มีในสต็อกสำหรับตัวแปร {{count}} รายการ",
+  "product-table-inventory-in-stock-count_other": " มีในสต็อกสำหรับตัวแปร {{count}} รายการ",
+  "reservation-form-location": "ที่ตั้ง",
+  "reservation-form-choose-where-you-wish-to-reserve-from": "เลือกสถานที่ที่คุณต้องการจองจาก",
+  "reservation-form-item-to-reserve": "สินค้าที่ต้องจอง",
+  "reservation-form-select-the-item-that-you-wish-to-reserve": "เลือกรายการที่คุณต้องการจอง",
+  "reservation-form-item": "รายการ",
+  "reservation-form-in-stock": "มีสินค้า",
+  "reservation-form-available": "มีอยู่",
+  "reservation-form-reserve": "จอง",
+  "reservation-form-remove-item": "ลบรายการ",
+  "reservation-form-description": "คำอธิบาย",
+  "reservation-form-what-type-of-reservation-is-this": "นี่คือการจองประเภทไหน?",
+  "reservations-table-reservations": "การจอง",
+  "reservations-table-edit": "แก้ไข",
+  "reservations-table-delete": "ลบ",
+  "reservations-table-confirm-delete": "คุณแน่ใจหรือไม่ว่าต้องการลบการจองนี้",
+  "reservations-table-remove-reservation": "ลบการจอง",
+  "reservations-table-reservation-has-been-removed": "การจองถูกลบออกแล้ว",
+  "new-success": "สำเร็จ",
+  "new-successfully-created-reservation": "สร้างการจองเรียบร้อยแล้ว",
+  "new-error": "ข้อผิดพลาด",
+  "new-cancel": "ยกเลิก",
+  "new-save-reservation": "บันทึกการจอง",
+  "new-reserve-item": "รายการจอง",
+  "new-metadata": "Metadata",
```

Comment:
`เมทาดาตา` or `ข้อมูลอภิพันธุ์` the latter one though it's correct by dictionary but it wouldn't make much sense for most of users.

The first one is reference from a wording used in [DGA Standard](https://standard.dga.or.th/glossary/%E0%B8%A1%E0%B8%B2%E0%B8%95%E0%B8%A3%E0%B8%90%E0%B8%B2%E0%B8%99%E0%B9%80%E0%B8%A1%E0%B8%97%E0%B8%B2%E0%B8%94%E0%B8%B2%E0%B8%95%E0%B8%B2/)

Reply:
I agree with you that it wouldn't make sense for most users if they use "เมทาดาตา" or "ข้อมูลอภิพันธุ์"

So, in this context, I think it can be transliterated as the same.

What do you think?